core/src/templates/components/site-search/site-search.twig

```diff
@@ -21,10 +22,13 @@
  *
  */
 #}
+{%- if search_input_id is empty -%}
+  {%- set search_input_id = search_input_name -%}
+{%- endif -%}
```

Comment:
This keeps it backwards compatible but we should remove it in version 7 and just go with the two separate ids.